lib/fog/storage/openstack.rb

```diff
@@ -43,7 +43,22 @@ class OpenStack < Fog::Service
       request :post_set_meta_temp_url_key
       request :public_url
 
+      module Utils
+        def require_mime_types
+          # Use mime/types/columnar if available, for reduced memory usage
+          require 'mime/types/columnar'
+          rescue LoadError
+            begin
+              require 'mime/types'
+            rescue LoadError
+              Fog::Logger.warning("'mime-types' missing, please install and try again.")
+              exit(1)
+            end
+          end
```

Comment:
I can't satisfy every piece of the hound.

module
  def
    rescue
      rescue
      end
    end
end

With 4 openings, and the implicit "begin" in def, I only need 3 endings.  I can't align them to all of them.  I can align to the def or the first rescue, what's your preference?